src/styles/colors.tsx

```diff
@@ -1,22 +1,17 @@
 // Designer Created Figma Colors
 // from https://www.figma.com/design/erFfzHvSTm5g1sjK6jWyEH/Working-Design-System?node-id=2100-4881&node-type=frame&t=vKGGXrs3Torz7kFE-0
 enum Colors {
-  // black & white
-  black = '#2E3338',
+  // primary beefy colors
   white = '#FFFFFF',
-
-  // grays
-  gray5 = '#505050',
-  gray4 = '#666666',
-  gray3 = '#757575',
-  gray2 = '#E6E6E6',
-  gray1 = '#F8F9F9',
-
-  // primary
+  darkBlue = '#242843',
   primary = '#2E3338',
-  accent = '#1AB775',
+  accent = '#4EB258',
+  lightBlue = '#B7B9C9',
+  blue200 = '#2D3154',
+  blue100 = '#363B63',
```

Comment:
This is where all of the interesting stuff happened. I used the `Rename Symbol` feature in Vscode to rename the enum values. The rest of the 300 files touched are just variable names changing and snapshots changing.